save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -230,10 +232,29 @@ class DownloadProjectController(private val configProperties: ConfigProperties,
                     log.error("Error to update test with url=$testSuiteUrl, path=$testSuitePaths")
                 }
                 .collect(Collectors.toList())
```

Comment:
This `collect` should be not needed now, because all elements will be collected on the outside Flux

Reply:
Removed